Cleaning of youthmappers.svg

jbelien opened the pull request:
I'm not sure why the file is in both `dist/img/` (where it should be with all the others) and `svg/` ...

bhousel commented:
The `svg/youthmappers.svg` can probably go away.
It doesn't look like you changed anything in these commits - what were you trying to do?

jbelien commented:
I removed the unnecessary `<?xml version="1.0" encoding="UTF-8"?>`
But more important, I also removed the `width="385px" height="385px"` ; it messes up any proper use of the SVG.

bhousel commented:
oh ok!
